docs/dev/coding_standards.rst

```diff
@@ -97,9 +97,86 @@ docstrings, and before module globals and constants.
+#. `Deprecation warning (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#deprecation-warning>`_
+#. `Extended Summary <https://numpydoc.readthedocs.io/en/latest/format.html#extended-summary>`_
+#. `Parameters (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#parameters>`_
+#. `Other Parameters (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#other-parameters>`_
+#. `Raises (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#raises>`_
+#. `See Also (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#see-also>`_
+#. `Notes (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#notes>`_
+#. `References (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#references>`_
+#. `Examples <https://numpydoc.readthedocs.io/en/latest/format.html#examples>`_
+
+Methods
+"""""""
+
+Example:
+.. code-block:: python
+
+    # This is a simple documentation example
+
+Sections:
+
+#. `Short summary <https://numpydoc.readthedocs.io/en/latest/format.html#short-summary>`_
+#. `Deprecation warning (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#deprecation-warning>`_
+#. `Extended Summary <https://numpydoc.readthedocs.io/en/latest/format.html#extended-summary>`_
+#. `Parameters (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#parameters>`_
+#. `Other Parameters (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#other-parameters>`_
+#. `Raises (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#raises>`_
+#. `See Also (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#see-also>`_
+#. `Notes (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#notes>`_
+#. `References (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#references>`_
+#. `Examples <https://numpydoc.readthedocs.io/en/latest/format.html#examples>`_
+
+
+More sections necessary for generators/etc. please see the original numpy standards 😄
+
+Modules
+"""""""
+
+Example:
+.. code-block:: python
+
+    # This is a simple documentation example
+
+Sections:
+
+#. `Short summary <https://numpydoc.readthedocs.io/en/latest/format.html#short-summary>`_
+#. `Extended summary (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#extended-summary>`_
+#. `Routine listings (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#routine-listings>`_
+#. `See also (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#see-also>`_
+#. `Notes (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#notes>`_
+#. `References (optional) <https://numpydoc.readthedocs.io/en/latest/format.html#references>`_
+#. `Examples <https://numpydoc.readthedocs.io/en/latest/format.html#examples>`_
+
 Internal Style Standards
 -------------------------
 
+Imports Shouldn't Be Buried Without a Reason
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+=======
```

Comment:
I think that is from a merge conflict, and should probably be removed.  Not sure why the RST linting didn't fail?

Reply:
Hm, not sure? I fixed it - unsure why it didn't flag it as a conflict on my end